src/core/qgsjsonutils.cpp

```diff
@@ -119,6 +120,12 @@ QString QgsJsonExporter::exportFeature( const QgsFeature &feature, const QVarian
     if ( mIncludeAttributes )
     {
       QgsFields fields = mLayer ? mLayer->fields() : feature.fields();
+      // List of formatters through we want to pass the values
+      QStringList formattersWhiteList;
+      formattersWhiteList << QStringLiteral( "KeyValue" )
```

Comment:
do we were moving back to use QString instead of QStringLiteral?

Reply:
Since when?

Reply:
probably I'm confused